infrastructure/template.yaml

```diff
@@ -1837,6 +1837,61 @@ Resources:
       Threshold: 0
       TreatMissingData: missing
 
+  AuditEventExceededAgeLimitAlarm:
```

Comment:
Should this be on the SQS queue not EventBridge and EventBridge to have it's own set of alarms?

Reply:
Caitlin raised the same thing, there is a branch in ipv-cri-common-infrastructure